### ISSUE TITLE
Can it be used for other generations?

### ISSUE THREAD

**Issue by OasisOfDragons** [Dabomstew/universal-pokemon-randomizer#107]
Can this be used for new games, such as X Y, ORAS, SM and USUM?

**Comment by RCTMASTA** [comment]
no, and I doubt Dabomstew has plans to, because you can already use pk3ds for that.
Download the latest nightly [here](https://teamcity.projectpokemon.org/viewLog.html?buildTypeId=Pk3DS_BuildWindows&buildId=lastSuccessful&buildBranch=%3Cdefault%3E), or the latest stable [here](https://github.com/kwsch/pk3DS/releases/latest)
